atlassian_jwt_auth/contrib/django/utils.py

```diff
@@ -0,0 +1,26 @@
+from django.conf import settings
+from jwt.exceptions import InvalidIssuerError
+
+
+def parse_jwt(verifier, encoded_jwt):
+    """Decode an encoded JWT using stored config."""
+    claims = verifier.verify_jwt(
+        encoded_jwt,
+        settings.ASAP_VALID_AUDIENCE,
+        leeway=60
+    )
+
+    return claims
+
+
+def verify_issuers(asap_claims, issuers):
+    """Verify that the issuer in the claims is valid and is expected"""
+    if asap_claims.get('iss') not in issuers:
```

Comment:
The flask code allows `issuers` to be an empty set or an empty list, this code should allow the same.